NARFO_BE/NARFO_BE/Models/Activity.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace NARFO_BE.Models
+{
+    public partial class Activity
+    {
+        public int ActiviryId { get; set; }
+        public string ActivityType { get; set; }
+        public string MemNo { get; set; }
+        public DateTime? ActivityDate { get; set; }
+        public string ActivityDescription { get; set; }
+        public DateTime? DateSubmitted { get; set; }
+        public string FirearmType { get; set; }
+        public string Caliber { get; set; }
+        public short? PostalShoot { get; set; }
+        public double? Score { get; set; }
+
+        public virtual Member MemNoNavigation { get; set; }
```

Comment:
please use proper words that describe exactly what this is. this will be easier for people to understand later. MemNoNavigation is not necessarily hard to decipher but this is an example of unnecessarily making something complicated for your self. because if you look at your model you have a double standard.
should **ActivityDescription**  be ActivityDescr
should **MemNo ** be MemberNumber?
stick to one naming convention please

Reply:
also spelling 

> ActiviryId